src/test/java/teammates/test/cases/ui/browsertests/AdminActivityLogPageUiTest.java

```diff
@@ -57,6 +57,12 @@ public void testViewActionsLink(){
              * This can happen if this test is run right after the server is started.
              * In this case, no view actions can be done.
              */
+        } catch (IndexOutOfBoundsException invisibletmtlogs) {
+            /*
+             * This can happen if all the log entries are from test accounts
+             * (i.e emails ending with .tmt) because they are invisible.
+             * In this case, no view actions can be done.
+             */
```

Comment:
I would go with the more informative ones, length should not be an issue. Wait a minute.